src/pages/login/CertificateGenerate.tsx

```diff
@@ -142,15 +142,18 @@ const CertificateGenerate: FC = () => {
                 <Col size={3}>
                   <h2 className="p-stepped-list__title p-heading--5">Trust</h2>
                 </Col>
-                <Col size={6}>
+                <Col size={8}>
                   <div className="p-stepped-list__content">
                     <p>
                       Download <code>lxd-ui.crt</code> and add it to the LXD
```

Comment:
There's another `lxd-ui.crt` here.

Reply:
missed this somehow, thanks for the catch!